test/smtcallback.ts

```diff
@@ -1,250 +1,250 @@
-      pragmaSMT = 'pragma experimental SMTChecker;\n';
+      pragmaSMT = 'pragma experimental SMTChecker;\n'
     } else {
-      settings = { modelChecker: { engine: 'all' } };
+      settings = { modelChecker: { engine: 'all' } }
     }
 
-    var input = { 'a': { content: preamble + pragmaSMT + 'contract C { function f(uint x) public pure { assert(x > 0); } }' } };
-    var inputJSON = JSON.stringify({
+    const input = { a: { content: preamble + pragmaSMT + 'contract C { function f(uint x) public pure { assert(x > 0); } }' } }
+    const inputJSON = JSON.stringify({
       language: 'Solidity',
       sources: input,
       settings: settings
-    });
+    })
 
-    var tests;
+    let tests
     if (!semver.gt(solc.semver(), '0.6.8')) {
       // Up to version 0.6.8 there were no embedded solvers.
       tests = [
         { cb: satCallback, expectations: ['Assertion violation happens here'] },
         { cb: unsatCallback, expectations: [] },
         { cb: errorCallback, expectations: ['BMC analysis was not possible'] }
-      ];
+      ]
     } else if (!semver.gt(solc.semver(), '0.6.12')) {
       // Solidity 0.6.9 comes with z3.
       tests = [
         { cb: satCallback, expectations: ['Assertion violation happens here'] },
         { cb: unsatCallback, expectations: ['At least two SMT solvers provided conflicting answers. Results might not be sound.'] },
         { cb: errorCallback, expectations: ['Assertion violation happens here'] }
-      ];
+      ]
     } else {
       // Solidity 0.7.0 reports assertion violations via CHC.
       tests = [
         { cb: satCallback, expectations: ['Assertion violation happens here'] },
         { cb: unsatCallback, expectations: ['Assertion violation happens here'] },
         { cb: errorCallback, expectations: ['Assertion violation happens here'] }
-      ];
+      ]
     }
 
-    for (var i in tests) {
-      var test = tests[i];
-      var output = JSON.parse(solc.compile(
+    for (const i in tests) {
+      const test = tests[i]
+      const output = JSON.parse(solc.compile(
         inputJSON,
         { smtSolver: test.cb }
-      ));
-      var errors = collectErrors(output);
-      st.ok(expectErrors(errors, test.expectations));
+      ))
+      const errors = collectErrors(output)
+      st.ok(expectErrors(errors, test.expectations))
     }
-    st.end();
-  });
+    st.end()
+  })
 
   t.test('Solidity smtCheckerTests', function (st) {
-    var testdir = path.resolve(__dirname, 'smtCheckerTests/');
+    const testdir = path.resolve(__dirname, 'smtCheckerTests/')
     if (!fs.existsSync(testdir)) {
-      st.skip('SMT checker tests not present.');
-      st.end();
-      return;
+      st.skip('SMT checker tests not present.')
+      st.end()
+      return
     }
 
     if (smtsolver.availableSolvers === 0) {
-      st.skip('No SMT solver available.');
-      st.end();
-      return;
+      st.skip('No SMT solver available.')
+      st.end()
+      return
```

Comment:
That's also something I wouldn't lump into the same PR. We even have a separate issue for it: #66.